modular_skyrat/modules/neural-lacing/code/implanter.dm

```diff
@@ -0,0 +1,27 @@
+/obj/item/autosurgeon/organ/corticalstack
+	desc = "A single use autosurgeon that contains a cortical stack. A screwdriver can be used to remove it, but implants can't be placed back in."
+	uses = 1
+	starting_organ = /obj/item/organ/corticalstack
+
+/obj/item/organ/corticalstack/syndicate
+	name = "Blood-Red Cortical Stack"
+	desc = "A strange, crystalline storage device containing 'DHF', digitised conciousness. This one has after-market modifications."
+	invasive = 0
+
+/obj/item/autosurgeon/organ/syndicate/corticalstack
+	desc = "A single use autosurgeon that contains a cortical stack. A screwdriver can be used to remove it, but implants can't be placed back in."
+	uses = 1
+	starting_organ = /obj/item/organ/corticalstack/syndicate
+
+/datum/uplink_item/device_tools/corticalstack
+	name = "Blood-Red Cortical Stack"
+	desc = " A marvel of modern technology; the cortical stack, For when you really want to avoid that death. This one can be removed without killing the user; allowing 'double-sleeving'."
```

Comment:
```suggestion
	desc = " A marvel of modern technology, the cortical stack — for when you really want to avoid that death. This one can be removed without killing the user, allowing 'double sleeving."
```